src/app/interface/search.interface.ts

```diff
@@ -167,7 +162,7 @@ export interface DatasetInfoInterface {
         name: string;
     };
     lastUpdatedAt?: string;
-    latestMetadataBlock?: MetadataBlockFragment;
+    latestMetadataBlock?: (MetadataBlockExtended | MetadataBlockFragment | any);
```

Comment:
This variable is assigned to

```
@Input() public latestMetadataBlock?: MetadataBlockFragment;
```
in overview-component.html:

```
            <app-dynamic-table
                ...
                [latestMetadataBlock]="datasetInfo.latestMetadataBlock"
            ></app-dynamic-table>
```            

which does not support `MetadataBlockExtended`. 

Somehow the code compiles if we leave this `any`, but it does not seem necessary.